tests/unit/parsers/test.manifestjson.js

```diff
@@ -2736,4 +2739,189 @@ describe('ManifestJSONParser', () => {
       return testHomepageUrl('http://test.org', true);
     });
   });
+
+  describe('@mdn/browser-compat-data: manifest properties', () => {
+    const usesArrayValueForFirefoxDesktopCompat = {
+      // There is a gap in the BCD data between two very old versions so we
+      // ignore it.
+      // Note: this might not be true for all props so we have to check the BCD
+      // data in case a new prop breaks one of the test cases below.
```

Comment:
Nit, may be worth to explicitly state what we plan to do in case of future failures triggered by this test (e.g. something like `... breaks one of the test cases below. In case of failure we want to double-check if we would need to special case the new cases or just add them to the list of expected ones that follows.`).